packages/schemas/src/types/user.ts

```diff
@@ -25,6 +26,17 @@ export type UserInfo = z.infer<typeof userInfoGuard>;
 
 export const userProfileResponseGuard = userInfoGuard.extend({
   hasPassword: z.boolean().optional(),
+  verifications: z
+    .array(
+      // Zod discriminatedUnion is too complicated for swagger
+      z.object({
+        ...baseVerification,
+        type: z.nativeEnum(MfaFactor),
+        agent: z.string().optional(),
+        used: z.boolean().optional(),
+      })
+    )
+    .optional(),
```

Comment:
should not be optional?

Reply:
Currently, only some endpoints need this field.

Reply:
updated with new endpoint